packages/appeals-service-api/src/mappers/questionnaire-submission/has-mapper.js

```diff
@@ -0,0 +1,87 @@
+class HasQuestionnaireMapper {
+	mapToPINSDataModel(journeyResponse) {
+		return [
+			{
+				questionnaire: {
+					LPACode: journeyResponse.LPACode,
+					caseReference: journeyResponse.caseReference,
+					isAppealTypeAppropriate: journeyResponse.answers['correct-appeal-type'] ? true : false,
+					doesTheDevelopmentAffectTheSettingOfAListedBuilding: journeyResponse.answers[
+						'affects-listed-building'
+					]
+						? true
+						: false,
+					affectedListedBuildings: () => {
+						if (journeyResponse.answers['affects-listed-building']) {
+							return journeyResponse.answers['listed-building-number'];
+						}
+					},
+					siteWithinGreenBelt: journeyResponse.answers['green-belt'] ? true : false,
+					howYouNotifiedPeople: journeyResponse.answers['notification-method'] ? true : false,
+					doesSiteHaveHealthAndSafetyIssues: journeyResponse.answers['safety-risks'] ? true : false,
+					healthAndSafetyIssuesDetails: () => {
+						if (journeyResponse.answers['safety-risks']) {
+							return journeyResponse.answers['affects-listed-building'];
```

Comment:
We might have to wait on the safety issues question to be added to get the name of the conditional field to use here